### PR TITLE
make Croissant support official, update exporter docs

### LINKED ISSUE #10341: Croissant support 🥐
Details to come. For now, please see:

- https://github.com/mlcommons/croissant/issues/530

### PR DESCRIPTION
**Pull request by pdurbin**
**What this PR does / why we need it**:

We have released to Maven Central an external metadata exporter from this new dedicated repo: https://github.com/gdcc/exporter-croissant

This pull request improves the guides to talk about Croissant and better explain all about external exporters. An inventory has been added.

This PR also adds docs about the behavior of the `<head>` tag that we changed with this pull request:

- #10382

**Which issue(s) this PR closes**:

- Closes #10341

**Special notes for your reviewer**:

A related PR to mention the new exporter:

- https://github.com/gdcc/dataverse-exporters/pull/4

**Suggestions on how to test this**:

Test the Croissant jar.

**Does this PR introduce a user interface change? If mockups are available, please link/include them here**:

Yes, when enabled, Croissant will appear in the dropdown of exports:

Screenshot-2024-03-14-at-11 51 42-AM

**Is there a release notes update needed for this change?**:

Included.

**Additional documentation**:

This PR is mostly docs. Some decent entry points:

- https://dataverse-guide--10533.org.readthedocs.build/en/10533/installation/advanced.html#external-metadata-exporters
- https://dataverse-guide--10533.org.readthedocs.build/en/10533/admin/discoverability.html#schema-org-head
- https://dataverse-guide--10533.org.readthedocs.build/en/10533/installation/advanced.html#croissant-croissant

### CODE REVIEW

File: doc/sphinx-guides/source/installation/config.rst [hunk 1]
```diff
@@ -3109,12 +3109,19 @@ Can also be set via any `supported MicroProfile Config API source`_, e.g. the en
...
+
+This JVM option is used to configure the file system path where external exporter JARs should be loaded from.
 
 ``./asadmin create-jvm-options '-Ddataverse.spi.exporters.directory=PATH_LOCATION_HERE'``
 
-If this value is set, Dataverse will examine all JARs in the specified directory and will use them to add, or replace existing, metadata export formats.
-If this value is not set (the default), Dataverse will not use external Exporters.
+If this value is set, Dataverse will examine all JARs in the specified directory and will use them to add new metadata export formats or (if the machine-readable name used in :ref:`export-dataset-metadata-api` is the same) replace built-in metatadata export formats.
+
+If this value is not set (the default), Dataverse will load any external exporters.
```
Comment (qqmyers) [inline doc/sphinx-guides/source/installation/config.rst:3120]: will not

Comment (pdurbin) [reply doc/sphinx-guides/source/installation/config.rst]: Whoops! Fixed in 844a08e716. Good catch. Thanks!

**Comment by pdurbin** [comment]
Yesterday at standup I mentioned some tweaks I was working on for the Croissant exporter and I did put out a release, 0.1.2:

https://github.com/gdcc/exporter-croissant/blob/croissant-0.1.2/CHANGELOG.md#012

**Comment by pdurbin** [comment]
At tech hours today I gave a preview of the talk I'm giving tomorrow for the Croissant Task Force. You can find my slides linked from this invitation to the community to join: https://groups.google.com/g/dataverse-community/c/JI8HPgGarr8/m/ONqgdyKJAAAJ

**Review by qqmyers — APPROVED** [review]
Looks good. I fixed some bad grammar that was in the text before the PR.

**Comment by pdurbin** [comment]
At standup today I mentioned I wanted to tweak this PR to remove the inventory of exporters from Sphinx now that we are keeping the list at https://github.com/gdcc/dataverse-exporters . I'm linking there instead now. See b5e7401.

I'm happy to answer any questions about this PR (just docs) and the code itself, which is at https://github.com/gdcc/exporter-croissant

**Pull request merged**